### PR TITLE
Add test for translations fix

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: karma/flows/test_controllers.coffee [hunk 1]
```diff
@@ -187,6 +187,9 @@ describe 'Controllers:', ->
         expect(modalScope.languages.from).toBe('eng')
         expect(modalScope.languages.to).toBe('ara')
 
+        # test we submit without error
+        modalScope.ok()
+
```
Comment (ericnewcomer) [inline karma/flows/test_controllers.coffee]: I presume this failed before the fix to only do USSD translations for USSD flows? Did you confirm it failed first?

Also, would be nicer to change the translation tests to use a helper method much like the editRules method that is shared throughout this test.

Comment (norkans7) [inline karma/flows/test_controllers.coffee]: right that failed without the changes, I'll add the helper method

**Comment by ericnewcomer** [comment]
Testing slack integration

File: karma/flows/test_controllers.coffee (hunk 1 above)
Comment (norkans7) [inline karma/flows/test_controllers.coffee]: @ericnewcomer this it updated

**Pull request merged**